frontend/src/patientApp/timeOfTest/PatientFormContainer.tsx

```diff
@@ -47,20 +54,77 @@ const PatientFormContainer = () => {
     );
   }
 
+  const savePerson = async (person: Nullable<PersonFormData>) => {
+    const {
+      street,
+      streetTwo,
+      city,
+      state,
+      county,
+      zipCode,
+      ...withoutAddress
+    } = person;
+    const updatedPatientFromApi = await PxpApi.updatePatient(
```

Comment:
Recommend a try/catch here for better error handling

Reply:
I don't know how the patient experience errors are handled but I think it is best to leave it as the same behavior as before. given the issue with the duplicate toasts